projects/common/ctpnxe/ctpnxe_system_bd_rx.tcl

```diff
@@ -0,0 +1,318 @@
+sbp_connect_interface_net "$project_name/gpio0_inst/INTR" \
+  "$project_name/cpu0_inst/IRQ_S4"
+sbp_connect_interface_net "$project_name/gpio1_inst/INTR" \
+  "$project_name/cpu0_inst/IRQ_S5"
+
+sbp_assign_addr_seg -offset 'h10004000 "$project_name/axi_apb2_inst/APB3_M" \
+  "$project_name/gpio1_inst/APB_S0"
+sbp_assign_addr_seg -offset 'h10003000 "$project_name/axi_apb1_inst/APB3_M" \
+  "$project_name/gpio0_inst/APB_S0"
+sbp_assign_addr_seg -offset 'h10002000 "$project_name/axi_apb0_inst/APB3_M" \
+  "$project_name/i2c0_inst/APB_S0"
+sbp_assign_addr_seg -offset 'h10000000 "$project_name/axi_ahb0_inst/AHBL_M" \
+  "$project_name/spi0_inst/AHBL_S0"
+sbp_assign_addr_seg -offset 'h00000000 "$project_name/cpu0_inst/LOCAL_BUS_M_DATA" \
+  "$project_name/tcm0_inst/LOCAL_BUS_IF_S0"
+sbp_assign_addr_seg -offset 'h00200000 "$project_name/cpu0_inst/AXI_M_INSTR" \
+  "$project_name/sysmem0_inst/AXI_S0"
+sbp_assign_addr_seg -offset 'h00000000 "$project_name/cpu0_inst/LOCAL_BUS_M_INSTR" \
+  "$project_name/tcm0_inst/LOCAL_BUS_IF_S1"
+
+## You can use this part in your project to instantiate a timer ip with max
+# 8 timers if you need it.
+
+# adi_ip_update -vlnv {latticesemi.com:ip:axi_interc0:1.2.0} \
+#   -meta_vlnv {latticesemi.com:ip:axi_interconnect:1.2.0} \
+#   -cfg_value {
+#     TOTAL_EXTMAS_CNT: 1,
+#     TOTAL_EXTSLV_CNT: 5,
+#     EXT_MAS_AXI_ID_WIDTH: 1,
+#     EXT_SLV_AXI_ID_WIDTH: 8
+#   } \
+#   -ip_iname "axi_interc0_inst"
+
+# adi_ip_instance -vlnv {latticesemi.com:ip:axi_apb3:1.1.0} \
+#   -meta_vlnv {latticesemi.com:ip:axi2apb_bridge:1.1.0} \
+#   -cfg_value {} \
+#   -ip_iname "axi_apb3_inst"
+
+# adi_ip_instance -vlnv {latticesemi.com:ip:timer0:1.3.0} \
+#   -meta_vlnv {latticesemi.com:ip:gp_timer:1.3.0} \
+#   -cfg_value {
+#     t1_cnt_up: count-up,
+#     T1_PERIOD_WIDTH: 32,
+#     t1_period_val: 0,
+#     t1_dis_pscaler: false,
+#     t2_cnt_up: count-up,
+#     T2_PERIOD_WIDTH: 32,
+#     t2_period_val: 0,
+#     t2_dis_pscaler: false,
+#     t3_cnt_up: count-up,
+#     T3_PERIOD_WIDTH: 32,
+#     t3_period_val: 0,
+#     t4_cnt_up: count-up,
+#     T4_PERIOD_WIDTH: 32,
+#     t4_period_val: 0
+#   } \
+#   -ip_iname "timer0_inst"
+
+# sbp_connect_net -name [sbp_get_nets -from $project_name/pll0_inst *clkop*] \
+#   "$project_name/timer0_inst/clk_i" \
+#   "$project_name/axi_apb3_inst/aclk_i"
+
+# sbp_connect_net -name [sbp_get_nets -from $project_name/cpu0_inst *resetn*] \
+#   "$project_name/timer0_inst/rst_n_i" \
+#   "$project_name/axi_apb3_inst/aresetn_i"
+
+# sbp_connect_interface_net "$project_name/axi_interc0_inst/AXI_M04" \
+#   "$project_name/axi_apb3_inst/AXI4_S"
+
+# sbp_connect_interface_net "$project_name/axi_apb3_inst/APB3_M" \
+#   "$project_name/timer0_inst/APB_S0"
+
+# sbp_connect_interface_net "$project_name/timer0_inst/INTR" \
+#   "$project_name/cpu0_inst/IRQ_S6"
+
+# sbp_assign_addr_seg -offset 'h10005000 "$project_name/axi_apb3_inst/APB3_M" \
+#   "$project_name/timer0_inst/APB_S0"
```

Comment:
Same as above, regarding commented code

Reply:
I need to check the timer requirements in our no-OS platform drivers to be able to decide. It will come soon, because I have to adopt the platform drivers to be able to use our DMA and SPI Engine drivers, but it may take some time. Until that I can remove this section.

Reply:
Solved in this  [commit](https://github.com/analogdevicesinc/hdl/pull/1270/commits/05015858329de4d1b632e5e6cdc0e36c9f6550c4).